src/components/Avatar.vue

```diff
@@ -0,0 +1,49 @@
+<!-- eslint-disable-next-line vue/no-v-html -->
+<template>
+  <img
+    v-if="!error"
+    :src="avatar"
+    @error="error = true"
```

Comment:
I think this will cause to trigger jdenticon regenerating on every placeholder, tough might not be so much. However it will still make a doomed request (that will end of 404) from the browser. Guess it will be goot to have a fallback (like unexpected server issue), but I believe it is not good to have a main logic that is driven by errors.

Reply:
We check if the user has uploaded custom avatar anyways by making a request to the backend. 404 is the error code which is returned if none was found.

Reply:
Imho this should be `v-if="!avatar"`

Reply:
`@error` is a correct way of handling it IMO as it executes a state change in the component and renders the icon if the backend returns 404.

its not possible to use it as reactive property as the request is async.

Reply:
Might be having "avatar" now taking null will not trigger a new request but instantly switch to the identicon, which is acceptable for me. Tough still is hackish.

Reply:
You could also add  `loading="lazy"` to the img tag to save some resources